package.json

```diff
@@ -39,6 +39,9 @@
     "test:schemas": "mocha libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js --exit --bail",
     "update-versions": "yarn workspace botbuilder-repo-utils update-versions"
   },
+  "resolutions": {
+    "node-fetch": "2.6.7"
```

Comment:
It would be good to update all `package.json` files in the repo that include this dependency as well. This `resolutions` key only governs developers who clone _this_ repository, not developers that download our SDK packages.